### PR TITLE
DDF-2810 Upgrade GeoWebCache from 0.6 to 0.7.0

### PR DESCRIPTION
**Pull request by clockard**
#### What does this PR do?
Updates the geowebcache version
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
@bdeining 
#### Select at least one member from relevant component team(s) from below (at least one component team member needs to approve the PR).
[IO](https://github.com/orgs/codice/teams/IO)
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@jlcsmith
@kcwire

#### Any background context you want to provide?
Depends on https://github.com/codice/geowebcache-osgi/pull/2 getting merged first
#### What are the relevant tickets?
[DDF-2810](https://codice.atlassian.net/browse/)
#### Checklist:
- [ ] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: catalog/spatial/geowebcache/geowebcache-app/pom.xml [hunk 1]
```diff
@@ -26,7 +26,7 @@
     <name>DDF :: Spatial :: GeoWebCache App</name>
 
     <properties>
-        <gwc-server.version>0.6</gwc-server.version>
+        <gwc-server.version>0.7.0-SNAPSHOT</gwc-server.version>
```
Comment (kcwire) [inline catalog/spatial/geowebcache/geowebcache-app/pom.xml]: We should only depend on released version of these 3rd party things so it doesn't cause issues when we need to do a release.  I recommend using SNAPSHOT while you're working through the upgrade but once it is working we should do the 0.7.0 release and change to that version.

Comment (clockard) [inline catalog/spatial/geowebcache/geowebcache-app/pom.xml]: Good point. Fixed. We just can't merge this until the geo web cache stuff is released.

Comment (kcwire) [inline catalog/spatial/geowebcache/geowebcache-app/pom.xml]: sweet! - if you tag me on the GWC PR I can help get that deployed when you're ready.

Comment (clockard) [inline catalog/spatial/geowebcache/geowebcache-app/pom.xml]: Tagged you on the pr https://github.com/codice/geowebcache-osgi/pull/2

**Comment by clockard** [comment]
Manually verified that the ddf geoweb cache is using the new 0.7.0 version with GeoWebCache 1.9.1

**Pull request merged**